redcap_bridge/cli.py

```diff
@@ -3,44 +3,57 @@
 """
 
 from redcap_bridge.server_interface import download_records
+from redcap_bridge.server_elab_interface import download_experiment
 
 
 def main(command_line=None):
     # importing required modules
     import argparse
 
     # create a parser object
-    parser = argparse.ArgumentParser(description="A simple Python interface for RedCap")
+    parser = argparse.ArgumentParser(description="A simple Python interface for RedCap and elabftw")
 
     parser.add_argument(
         '--debug',
         action='store_true',
         help='Print debug info'
     )
 
-    subparsers = parser.add_subparsers(dest='command')
-    download = subparsers.add_parser('download', help='Downloads the data records')
```

Comment:
You removed the `download` option completetly, so `RedCapBridge download my_file.csv my_config.json` is not working any more with those modifications. For any user who has set up a script to run this command regularly, this will break. Could you ensure that all previous calls to the CLI work as before, but add another optional `--server` argument that indicates if you want to download from a redcap or an elab server?

Reply:
Also note the CLI tests here: https://github.com/INT-NIT/DigLabTools/blob/main/redcap_bridge/test_redcap/test_cli.py

Reply:
Oopss I did remove too many things, I made some changes, I think it should be fine now